_includes/tutorials/rekeying/ksql/code/test/output.json

```diff
@@ -4,72 +4,72 @@
       "topic": "ratings_keyed_by_id",
       "key": 294,
       "value": {
-        "id": 294,
-        "rating": 8.2
+        "ID": 294,
```

Comment:
Just for my own education, these changes here are for KSQL changes in output correct?

Reply:
It looks like expected test output was changed to match the same upper-casing as is applied when using the `PRINT` command.